### PR TITLE
Update branding to 3.1.2

### PR DESCRIPTION
**Pull request by JunTaoLuo**
waiting until branches are open.

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
Ready to go cc @dougbu @wtgodbe

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -13,49 +13,49 @@
...
-      <Sha>d40e21ccc14908a054b2181b1d6aeb22c49c630d</Sha>
+    <Dependency Name="Microsoft.Extensions.Logging" Version="3.1.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/aspnet-Extensions</Uri>
+      <Sha>d00c382ec5d68a85d2eb4a49ab4559b8db7a2390</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="3.1.1-servicing.19576.9" CoherentParentDependency="Microsoft.Extensions.Logging">
-      <Uri>https://github.com/dotnet/core-setup</Uri>
-      <Sha>f3f2dd583fffa254015fc21ff0e45784b333256d</Sha>
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="3.1.1" CoherentParentDependency="Microsoft.Extensions.Logging">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-core-setup</Uri>
```
Comment (wtgodbe) [inline eng/Version.Details.xml]: These URI changes seem wrong, & I guess this would happen in every repo - @mmitche do these need to be manually fixed?

Comment (mmitche) [reply eng/Version.Details.xml]: They are right.  That is where 3.1.1 was originally built.  They will update automatically over time as dependency flow happens.

Comment (JunTaoLuo) [reply eng/Version.Details.xml]: According to @mmitche this doesn't matter.

**Comment by JunTaoLuo** [comment]
Ready to go @dougbu @wtgodbe

**Pull request merged**